kernel/packages/shared/dao/reducer.ts

```diff
@@ -89,6 +90,17 @@ export function daoReducer(state?: DaoState, action?: AnyAction): DaoState {
           }
         })
       }
+    case MARK_CATALYST_REALM_CONNECTION_ERROR:
+      return {
+        ...state,
+        candidates: state.candidates.map(it => {
+          if (it.catalystName === action.payload.catalystName) {
+            return { ...it, layer: { ...it.layer, elapsed: Number.MAX_SAFE_INTEGER } }
```

Comment:
Shouldn't we use a status for this? I know we already used the users count to mark a layer as full, but that made much more sense since the response signaled that state.

I suppose this also makes sense, since you are marking the response time of a catalyst as "infinite", which would be equivalent to an error state. But somehow I feel this is a little hacky.

In any case, it should work, so I'll still approve the PR.

Reply:
Thanks for the feedback. I added server connection status to candidates. Please review 😬